detections/endpoint/mailsniper_invoke_functions.yml

```diff
@@ -0,0 +1,45 @@
+name: Mailsniper Invoke functions
+id: a36972c8-b894-11eb-9f78-acde48001122
+version: 1
+date: '2021-05-19'
+author: Teoderick Contreras, Splunk
+type: batch
+datamodel:
+- Endpoint
+description: This search is to detect known mailsniper.ps1 functions executed in a
+  machine. This technique was seen in some attacker to harvest some sensitive e-mail
+  in a compromised exchange server.
+search: '`powershell` EventCode=4104 Message IN ("*Invoke-GlobalO365MailSearch*",
```

Comment:
@t-contreras could you please try extending this to a couple of other events, same thing would appear there, but with them we'd ensure coverage:

in EventID 4103 - these same modules would appear in Message field
(see this work in SSA for an "event" sample: https://confluence.splunk.com/display/~tramamurthy/4103+and+4104+mappings+to+Endpoint_process+fields+and+new+fields+addedP

In even 4688: 
These modules would be listed in the Process Command Line field.

Reply:
@miskoSplunk thanks for this feedback. will take a look on those events and Ill implement that also if possible ;) thanks again